### ISSUE TITLE
Launch at Login does not work (Ice 0.5.0, Sonoma 14.3.1)

### ISSUE THREAD

**Issue by andriigorshunov** [jordanbaird/Ice#20]
Hello everybody!
Launch app at Login does not work. Each time I restart my machine, my menu bar resets and there are no signs of Ice to be launched. Then I launch manually and it starts to work. The tick in Settings (General -> Launch at Login) is active. I reactivated it couple times but it still does not work.

Ice 0.5.0, Sonoma 14.3.1, Macbook Pro 16 M1 MAX

**Comment by jordanbaird** [comment]
Could you check `System Settings > General > Login Items > Open at Login` to make sure Ice is there? It should be automatically added when `Launch at Login` is checked.

**Comment by andriigorshunov** [comment]
I just checked and it in settings. It was there but the app icon was white and after right-click, Finder could not locate it.
I deleted app and then then unzipped it again an moved to Application folder. Launched from there and now after restart it works correctly. Also under System Settings > General > Login Items > Open at Login, I can see app **with** Icon so it seems like MacOS could not locate app correctly before, even though it was visible as app in Launchpad. 

Looks like it is resolved now. Thank you.
